src/VirtualClient/VirtualClient.Actions/OpenSSL/OpenSslMetricsParser.cs

```diff
@@ -290,7 +290,24 @@ private bool TryParseSignVerifyPerformanceResults(out DataTable results)
             // Example:
             //                    sign verify    sign/s verify/s
             //  rsa 2048 bits 0.000820s 0.000024s   1219.7  41003.9
+
             MatchCollection rsaPerformanceResults = Regex.Matches(this.RawText, $@"((?:\w *\(*)+(?:bits|\)))(\s*[0-9\.]+s)(\s*[0-9\.]+s)(\s*[0-9\.]+)(\s*[0-9\.]+)", RegexOptions.IgnoreCase | RegexOptions.Singleline);
+           
+            IEnumerable<string> ecdhColumns = new List<string>()
+            {
+                "op",
+                "op/s"
+            };
+
+            // Example:
+            //                           op      op/s
+            // 448 bits ecdh(X448)   0.0000s  42896.0
+
+            if (rsaPerformanceResults?.Any() == false)
+            {
+                rsaPerformanceResults = Regex.Matches(this.RawText, $@"((?:\w *\(*)+(?:bits|\)))(\s*[0-9\.]+s)(\s*[0-9\.]+)", RegexOptions.IgnoreCase | RegexOptions.Singleline);
```

Comment:
Please add the ops in a separate method for consistency

Reply:
Done that, please review and let me know if it requires any more changes.